tgui/packages/tgui/interfaces/PrisonerImplantManager.js

```diff
@@ -76,7 +76,7 @@ export const PrisonerImplantManager = (props, context) => {
           {trackingInfo
             .map(implant => (
               <>
-                <Box bold>Subject: {implant.subject}</Box><Box key={implant.id}>
+                <Box bold>Subject: {implant.subject}</Box><Box key={implant.subject}>
```

Comment:
Why this change here?

Reply:
I removed the id var from the implant list.